app/src/main/java/org/oppia/app/topic/play/StorySummaryAdapter.kt

```diff
@@ -38,7 +40,20 @@ class StorySummaryAdapter(
     RecyclerView.ViewHolder(binding.root) {
     internal fun bind(storySummary: StorySummary, @Suppress("UNUSED_PARAMETER") position: Int) {
       binding.setVariable(BR.storySummary, storySummary)
-      binding.root.setOnClickListener {
+
+      val totalChapterCount = storySummary.chapterCount
+      val chapterSummaries = storySummary.chapterList
+      val completedChapterCount =
+        chapterSummaries.map(ChapterSummary::getChapterPlayState)
+          .filter {
+            it == ChapterPlayState.COMPLETED
+          }
+          .size
+
+      val storyProgressPercentage: Int = (completedChapterCount / totalChapterCount) * 100
```

Comment:
nit change: Please remove explicit type specifications

Reply:
Here we need this type to make sure that the result of right-side is always in integer.

Reply:
Will this division work correctly? It seems like an int divided by an int--won't this always result in 0? I think you need to convert to a float somewhere before converting back to int for display.